homeassistant/util/package.py

```diff
@@ -30,22 +30,30 @@ def is_docker_env() -> bool:
     return Path("/.dockerenv").exists()
 
 
-def is_installed(package: str) -> bool:
+def get_installed_versions(specifiers: set[str]) -> set[str]:
+    """Return a set of installed packages and versions."""
+    return {specifier for specifier in specifiers if is_installed(specifier)}
+
+
+def is_installed(requirement_str: str) -> bool:
```

Comment:
Probably need to move this fix into another PR as its only related in that I discovered it while testing